README.md

```diff
@@ -107,6 +107,8 @@ Options:
   -j, --parallel <number>     Run tests in parallel
   -J, --autoParallel          Run tests in parallel (automatically detect core count)
   --tmpDir <path>             Directory to test modules in
+  --withTags <tag>            Only test modules from the lookup that contain a matching tag field
+  --excludeTags <tag>         Specify which tags to skip from the lookup
```

Comment:
How do you specify multiple tags to run or exclude? Could you update the README and man pages to cover that?

Also there are no tests for multiple tags.